document/core/syntax/modules.rst

```diff
@@ -328,7 +328,7 @@ The |MSTART| component of a module declares the :ref:`function index <syntax-fun
 
 .. note::
    The start function is intended for initializing the state of a module.
-   The module and its exports are not accessible before this initialization has completed.
+   The module and its exports are not accessible to the host before this initialization has completed.
```

Comment:
Depending on the environment, it's not just the host, but other Wasm modules may be clients as well.
```suggestion
   The module and its exports are not accessible to clients before this initialization has completed.
```